src/core/cookie.js

```diff
@@ -139,6 +129,12 @@ DM.provide('Cookie',
      */
     clear: function()
     {
-        DM.Cookie.setRaw('', 0, DM.Cookie._domain);
+        var key = 'dms_' + DM._apiKey;
+        var domain = document.domain === 'localhost' ? false : DM.Cookie._domain;
```

Comment:
This line could be factorized with [L109](https://github.com/dailymotion/dailymotion-sdk-js/pull/29/files#diff-55e3118b2886288b4c4fc0a7833b4986R109).